messages/es.json

```diff
@@ -14,6 +14,9 @@
     "NAME": "Nombre",
     "USER_NAME": "Nombre de usuario",
     "NAME_TO_RECOGNIZE": "Nombre con el que te reconocerán",
+    "TYPE_OF_USER": "Tipo de usuario",
+    "CLIENT": "Cliente",
+    "BUSINESS": "Negocio",
```

Comment:
change BUSINESS to BUSINESS_OWNER and spanish too

Reply:
Done